test/apps/iodemo/io_demo.cc

```diff
@@ -661,20 +738,22 @@ static int parse_args(int argc, char **argv, options_t* test_opts)
             std::cout << "Supported options are:" << std::endl;
-            std::cout << "  -p <port>                TCP port number to use" << std::endl;
-            std::cout << "  -o <op1,op2,...,opN>     Comma-separated string of IO operations [read|write]" << std::endl;
-            std::cout << "                           NOTE: if using \"random\", performance" << std::endl;
-            std::cout << "                                 measurments may be inaccurate" << std::endl;
-            std::cout << "  -d <min>:<max>           Range that should be used to get data" << std::endl;
-            std::cout << "                           size of IO payload" << std::endl;
-            std::cout << "  -b <number of buffers>   Number of IO buffers to use for communications" << std::endl;
-            std::cout << "  -i <iterations-count>    Number of iterations to run communication" << std::endl;
-            std::cout << "  -w <window-size>         Number of outstanding requests" << std::endl;
-            std::cout << "  -r <io-request-size>     Size of IO request packet" << std::endl;
-            std::cout << "  -c <client retries>      Number of retries on client for failure" << std::endl;
-            std::cout << "  -t <client timeout>      Client timeout, in seconds" << std::endl;
-            std::cout << "  -s <random seed>         Random seed to use for randomizing" << std::endl;
-            std::cout << "  -v                       Set verbose mode" << std::endl;
+            std::cout << "  -p <port>                  TCP port number to use" << std::endl;
+            std::cout << "  -o <op1,op2,...,opN>       Comma-separated string of IO operations [read|write]" << std::endl;
+            std::cout << "                             NOTE: if using \"random\", performance" << std::endl;
+            std::cout << "                                   measurments may be inaccurate" << std::endl;
+            std::cout << "  -d <min>:<max>             Range that should be used to get data" << std::endl;
+            std::cout << "                             size of IO payload" << std::endl;
+            std::cout << "  -b <number of buffers>     Number of IO buffers to use for communications" << std::endl;
+            std::cout << "  -i <iterations-count>      Number of iterations to run communication" << std::endl;
+            std::cout << "  -w <window-size>           Number of outstanding requests" << std::endl;
+            std::cout << "  -r <io-request-size>       Size of IO request packet" << std::endl;
+            std::cout << "  -c <client retries>        Number of connection retries on client" << std::endl;
+            std::cout << "                             (or \"inf\") for failure" << std::endl;
+            std::cout << "  -t <client timeout>        Client timeout" << std::endl;
+            std::cout << "  -l <client run-time limit> Time limit to run the IO client (or \"inf\")" << std::endl;
```

Comment:
can you please add an example of how to set the time- limit?

Reply:
fixed